README.md

```diff
@@ -64,7 +59,7 @@ in the modes described below:
  unit that communicates with the cluster by relating to the
  `apache-hadoop-plugin` subordinate charm:
 
-    juju-quickstart apache-hadoop-spark
+    juju-quickstart apache-bigtop-processing-spark
```

Comment:
I think we settled on "bigtop-processing-X" for the bundle naming pattern, so this would be "bigtop-processing-spark".